lib/lhm.rb

```diff
@@ -57,6 +58,15 @@ def self.cleanup(run = false, options = {})
       trigger.respond_to?(:trigger) ? trigger.trigger : trigger
     end.select { |name| name =~ /^lhmt/ }
 
+    if options[:table_name]
+      table_name = "_#{options[:table_name]}"
+      lhm_tables.select! do |table|
+        stripped_table_name = table.starts_with?("lhma") ? table.slice(28..-1) : table.slice(4..-1)
```

Comment:
@ImAjayS In which we need this else part?

Reply:
There are couple of lhm prefixes - lhma, lhmn. lhma is used for post migration, and lhmn is used during migration(new table where migration is occuring). Will mention these in comments